aut_docs/Installation_Guide_WindowsOS.md

```diff
@@ -28,10 +27,9 @@
        pip install virtualenv
     ```
 - Download Git 
-    ```
-       [Git](https://git-scm.com/download/win)
-    ```
-     **Note:** While installing select the option **Use Git and optional Unix Tools from the Windows Command Prompt** for using Git from the command prompt
+  [Git](https://git-scm.com/download/win)
```

Comment:
Same here, remove this Git word and hyperlink the previous one to remove duplication of words